pytrack/project/py_track_project.py

```diff
@@ -12,13 +12,16 @@
 
 from pytrack.interface import DVCInterface
 import subprocess
+from datetime import datetime
 
 log = logging.getLogger(__file__)
 
 
 class PyTrackProject(DVCInterface):
```

Comment:
class doc string

Reply:
solved in 2175398a95c5813644aef5da0d41bc8ce2df9d21